Makefile

```diff
@@ -42,6 +42,7 @@ dist_SCRIPTS= \
 	init \
```

Comment:
This is not needed, see the other comment, because dynamically linked busybox should be handled properly.